core/src/main/java/org/apache/struts2/dispatcher/filter/StrutsPrepareFilter.java

```diff
@@ -43,6 +43,8 @@ public class StrutsPrepareFilter implements StrutsStatics, Filter {
     protected static final String REQUEST_EXCLUDED_FROM_ACTION_MAPPING = StrutsPrepareFilter.class.getName() + ".REQUEST_EXCLUDED_FROM_ACTION_MAPPING";
 
     protected PrepareOperations prepare;
+
+    @Deprecated
```

Comment:
Please document why it's deprecated and what to use instead. Also `since` should be added, and feel free to create a ticket in JIRA to remove this deprecated element in Struts 7. [An example deprecation](https://github.com/apache/struts/blob/master/core/src/main/java/org/apache/struts2/interceptor/FileUploadInterceptor.java#L176)

Reply:
Yep let me rectify
Regarding a ticket for the deletion - my assumption would be that all deprecated elements are deleted in the next major version. Do we still require individual follow-up issues in the case of a deprecation?

Reply:
We can have one general ticket to remove all the deprecated options, just to be sure we won't miss this one at some point (so `since` would be needed)

Reply:
Fixed and created this general issue - WW-5411